src/Microsoft.IdentityModel.Clients.ActiveDirectory/TokenCache.cs

```diff
@@ -392,7 +392,7 @@ internal static async Task<List<string>> GetOrderedAliasesAsync(string authority
             return aliasedAuthorities;
         }
 
-        internal AdalResultWrapper LoadFromCacheCommon(CacheQueryData cacheQueryData, RequestContext requestContext)
+        internal /* internal for test */ AdalResultWrapper LoadFromCacheCommon(CacheQueryData cacheQueryData, RequestContext requestContext)
```

Comment:
>/* internal for test */ [](start = 17, length = 23)

nit: is this comment needed?

Reply:
It's more so that ADAL developers don't call LoadFromCacheCommon from anywhere but this class. Normally this method should be private, but we need it to be internal so that tests can call it. 

Not the best design in terms of test hooks, but redesigning this class is not appropriate at this time in ADAL's lifecycle.